backends/cuda/runtime/cuda_backend.cpp

```diff
@@ -365,6 +467,10 @@ class ET_EXPERIMENTAL CudaBackend final
     delete handle;
     clear_all_tensors();
   }
+
+ private:
+  mutable std::mutex skip_copy_method_mutex_;
+  std::string skip_copy_method_;
```

Comment:
shouldn't it be an arrary of string cus we may skip copy on multiple methods? Its ok to be update in the following PRs.

Reply:
Yeah next PR will be supporting a comma separated string.